src/style.css

```diff
@@ -0,0 +1,109 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+}
+
+.block {
+  display: flex;
+}
+
+.card {
+  flex: 200px 0 0;
+  justify-content: center;
+  padding: 15px 14px;
+  background: #fff;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: 15px auto 45px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.card__code {
+  line-height: 13px;
+  margin-top: 5px;
+  margin-bottom: 15px;
+}
+
+.card__info {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.card__info--price {
+  align-items: flex-end;
+}
+
+.card__stars {
+  display: flex;
+}
+
+.card__star {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+}
+
+.card__stars--4 .card__star:nth-child(-n+4) {
```

Comment:
add classes from stars block task, so if I change the class of your element in dev tools I will see how the count of active stars changing.